app/test-utils/helper.js

```diff
@@ -11,12 +11,13 @@ const extractCsrfToken = (res) => {
   return $('[name=_csrf]').val();
 };
 
-export const getCsrfTokenFromGet = async (app, getPath) => {
+export const getCsrfTokenFromGet = async (app, getPath, authenticationCookie = undefined) => {
```

Comment:
won't it will be undefined already if it is not passed in?

Reply:
😂 what was I thinking